### PR TITLE
Otel 1.0 support

### PR DESCRIPTION
**Pull request by ricardoccpaiva**

### CODE REVIEW

**Comment by ricardoccpaiva** [comment]
@dvic @indrekj your comments are always welcome

File: mix.lock [hunk 1]
```diff
@@ -10,8 +10,8 @@
...
+  "opentelemetry": {:hex, :opentelemetry, "1.0.0", "6e98f4a9230681b2e4c88d45783ce1c02d671ffc0b5ac0cba69a34a3f5ada8d8", [:rebar3], [{:opentelemetry_api, "~> 1.0.0", [hex: :opentelemetry_api, repo: "hexpm", optional: false]}], "hexpm", "08d8697740f70594d05067cb62a0a8845ff568b2d47e1f8c78c46708ab58a74f"},
+  "opentelemetry_api": {:hex, :opentelemetry_api, "1.0.0", "6e501f750ead189f35aed07eb8023fa6655fca12f913a196102f67db4ed5172c", [:mix, :rebar3], [], "hexpm", "ac51520bde21fdea7f82cea9236ce4e88a21281c22bc23b0f1fa3b28b4352fcf"},
   "opentelemetry_telemetry": {:hex, :opentelemetry_telemetry, "1.0.0-beta.2", "b840eee9e68307ad7fa4ee316da19db3f8e30763b87737d3304782ca3cc296a2", [:mix, :rebar3], [{:opentelemetry_api, "~> 1.0.0-rc.1", [hex: :opentelemetry_api, repo: "hexpm", optional: false]}, {:telemetry, "~> 0.4", [hex: :telemetry, repo: "hexpm", optional: false]}, {:telemetry_registry, "~> 0.2.1", [hex: :telemetry_registry, repo: "hexpm", optional: false]}], "hexpm", "e8b12f42614d0aeb6a49001c75ca035544950f736fdbb240177838674f99e1e2"},
```
Comment (indrekj) [inline mix.lock:15]: 1.0.0-beta.7 is out

File: README.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ by adding `opentelemetry_tesla` to your list of dependencies in `mix.exs`:
 ```elixir
 def deps do
   [
-    {:opentelemetry_tesla, "~> 1.3.2-rc.1"}
+    {:opentelemetry_tesla, "~> 1.3.2-rc.3"}
```
Comment (indrekj) [inline README.md:28]: would be nice if we could get rid of `-rc` suffix, but I guess we cannot because opentelemetry_telemetry is still using beta version

**Review by indrekj — APPROVED** [review]

**Comment by indrekj** [comment]
I guess list also works, but could also change https://github.com/ricardoccpaiva/opentelemetry_tesla/blob/master/lib/opentelemetry_tesla.ex#L129 to use map

**Review by dvic — APPROVED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (ricardoccpaiva) [reply README.md:28]: exactly, not sure how we are going to manage this when this library moves to https://github.com/open-telemetry/opentelemetry-erlang-contrib

**Comment by ricardoccpaiva** [comment]
> I guess list also works, but could also change https://github.com/ricardoccpaiva/opentelemetry_tesla/blob/master/lib/opentelemetry_tesla.ex#L129 to use map

both data structures work, but yeah, going to change to a map.

File: mix.lock (hunk 1 above)
Comment (ricardoccpaiva) [reply mix.lock:15]: going to update all deps.

**Pull request merged**